tests/unit/io/test_eventletreactor.py

```diff
@@ -18,50 +18,35 @@
 except ImportError:
     import unittest # noqa
 
-from tests.unit.io.utils import submit_and_wait_for_completion, TimerCallback
-from tests import is_eventlet_monkey_patched
+from tests.unit.io.utils import TimerConnectionTests
+from tests.unit.io.eventlet_utils import restore_saved_module, eventlet_un_patch_all
+from tests import notpypy, MONKEY_PATCH_LOOP
 import time
+from eventlet import monkey_patch, kill
+import threading
 
 try:
     from cassandra.io.eventletreactor import EventletConnection
 except ImportError:
     EventletConnection = None  # noqa
 
-
-class EventletTimerTest(unittest.TestCase):
-
+class EventletTimerTest(unittest.TestCase, TimerConnectionTests):
+    # There are some issues with some versions of pypy and eventlet
+    @notpypy
     def setUp(self):
-        if EventletConnection is None:
-            raise unittest.SkipTest("Eventlet libraries not available")
-        if not is_eventlet_monkey_patched():
+        if not MONKEY_PATCH_LOOP:
```

Comment:
Yeah, I'll change this, but I'd rather leave the `Eventlet is None` comparison in case Eventlet is not installed